how-to-guides/mainnet.md

```diff
@@ -68,16 +70,16 @@ $$
 $$
 
 $$
-\text{Remaining shares: } 16382 \times 482 \text{ bytes}
+\text{Remaining shares: } 262,142 \times 482 \text{ bytes}
 $$
 
 $$
-\text{Total Bytes: } 7,896,602 \text{ bytes}
+\text{Total Bytes: } 126,352,922 \text{ bytes}
```

Comment:
this math seems accurate for Arabica but somewhere above we should clarify that this is only true for Arabica currently

Reply:
isn't 128mib == `134218`?